### DISCUSSION TITLE
A small feat request

### DISCUSSION THREAD

**Discussion by syntaxdot** [Truinto/DarkCodex#61] [Ideas]
Hey, Truinto! 
Thanks a lot for your work! 
Could you please add a feat that would allow spontaneous casters to apply metamagic to spells and keep the standard casting time?

**Comment by Truinto** [comment]
Can you give me the name, description, and prerequisites?

**Reply by syntaxdot** [reply]
Sure! 

Metamagic Adept
Prerequisites: Charisma 13, one metamagic feat, able to spontaneously cast spells.
Benefit: You can apply any metamagic feat to spells without increasing casting time.
